blog/models.py

```diff
@@ -81,6 +82,11 @@ def image(self):
         if len(self.images):
             return self.images[0]
 
+    def delete(self):
```

Comment:
I'll try on my side, if I get stuck on something too long, I'll say it to you :wink: